tools/StandardAnchorTags/TocSectionNumberBuilder.cs

```diff
@@ -166,6 +172,11 @@ private SectionLink BuildSectionLink(SectionHeader header, string filename)
             header.level = level;
 
             // A few cases for section number:
+
+            if (string.IsNullOrWhiteSpace(fields[1]))
+            {
+                throw null;
```

Comment:
Why *are* we throwing null here?

Reply:
Test code I meant to remove. (It helped me force the debugger to break on the right spot). Removing.

Reply:
Phew :)